core/src/main/scala/cats/data/OptionT.scala

```diff
@@ -287,6 +290,29 @@ private[data] trait OptionTMonad[F[_]] extends Monad[OptionT[F, ?]] {
     )))
 }
 
+private[data] trait OptionTMonadErrorMonad[F[_]] extends MonadError[OptionT[F, ?], Unit] {
```

Comment:
Wondering if it'd be possible to extend `OptionTMonad[F]` here to reduce duplication (like the trait below does).

Reply:
I see.. You are right. I should have mixin `OptionTMonad[F]`. Changing it..